public/sass/base/components/available_exercises_listing.scss

```diff
@@ -0,0 +1,52 @@
+@import '../variables';
+
+/***************
+
+  Component Name: Available Exercises Listing
+  Descrption: This component is used to provide styling for available exercises list.
+
+***************/
+
+.available_exercises_listing {
+  margin: 0;
+  padding: 0;
+  li {
+    list-style: none;
+    padding: 0 5px;
+    position: relative;
+    min-height: 35px;
+    &:not(:last-child) {
+      border-bottom: 1px solid #ccc;
+    }
+    a {
+      color: $exercism-red;
+      font-size: 14px;
```

Comment:
AFAIK, 14px is the normal font size. In my suggestion, the link to the exercise has a bigger font. That was on purpose. 😉